src/store/modules/verifymail.js

```diff
@@ -28,7 +28,7 @@ export const actions = {
   async verify ({ commit }, key) {
     commit(types.REQUEST)
```

Comment:
Could migrate to use meta status?

Reply:
Definitely, but I'd make it a separate PR 🚫